src/js/trove/load-lib.js

```diff
@@ -97,6 +97,16 @@
     function getResultProvides(mr) {
       return mr.val.provides;
     }
+    function getResultStackTrace(mr) {
+      var runtime = mr.val.runtime;
+      if (!runtime.isFailureResult(mr.val.result)) {
+        runtime.ffi.throwMessageException("Tried to get stacktrace of non-failing module result.");
+      }
+
+      var res = getModuleResultResult(mr);
+      var stackString = runtime.printPyretStack(res.exn.pyretStack);
+      return runtime.makeString(stackString);
```

Comment:
Would getting back a `raw-array` be more useful, split on new lines?  Just thinking about possible interfaces.